src/middleware/authHandler.js

```diff
@@ -0,0 +1,24 @@
+const logger = require('../config/logger');
+const jwt = require('jsonwebtoken');
+
+const config = require('../config/authConfig');
+const { StatusCodes } = require('http-status-codes');
+
+const handler = (req, res, next) => {
+    const token = req.headers[config.TOKEN_HEADER];
+    if (token) {
+        jwt.verify(token, config.SECRET, (err) => {
+            if (err) {
+                logger.error(`FORBIDDEN: bad token`);
+                res.status(StatusCodes.FORBIDDEN).end('Toke validation failed');
+            } else {
+                return next();
+            }
+        });
+    } else {
+        logger.error(`FORBIDDEN: no toke found`);
+        res.status(StatusCodes.FORBIDDEN).end('No token provided');
```

Comment:
здесь не forbidden (403), а 401 

```
In case of the HTTP Authorization header is absent in the request, 
the middleware should stop further controller method execution and return HTTP 401 code
 (Unauthorized Error) and standard error message.
```